mlflow/gateway/providers/anthropic.py

```diff
@@ -110,6 +227,77 @@ def __init__(self, config: RouteConfig) -> None:
         }
         self.base_url = "https://api.anthropic.com/v1/"
 
+    async def chat_stream(
+        self, payload: chat.RequestPayload
+    ) -> AsyncIterable[chat.StreamResponsePayload]:
+        payload = jsonable_encoder(payload, exclude_none=True)
+        self.check_for_model_field(payload)
+        stream = send_stream_request(
+            headers=self.headers,
+            base_url=self.base_url,
+            path="messages",
+            payload={
+                "model": self.config.model.name,
+                **AnthropicAdapter.chat_streaming_to_model(payload, self.config),
+            },
+        )
+
+        indices = []
+        metadata = {}
+        async for chunk in stream:
+            chunk = chunk.strip()
+            if not chunk:
+                continue
+
+            # No handling on "event" lines
+            prefix, content = chunk.decode("utf-8").split(":", 1)
+            if prefix != "data":
+                continue
```

Comment:
```suggestion
            prefix, content = chunk.split(b":", 1)
            if prefix != "data":
                continue
            content = content.decode("utf-8")
```

Is it possible to split `chunk` with `b":"`?

Reply:
sure